src/how_to.md

```diff
@@ -0,0 +1,24 @@
+# "How-to" guides
+
+Here you’ll find short answers to “How do I….?” types of questions. These how-to guides don’t cover topics in depth – you’ll find that material in the [Topic](./topics.md) and [Reference](./references.md) Guides. However, these guides will help you quickly accomplish common tasks.
+
+  - [Transpile to Javascript](./transpile_js.md)
+  - [Contribute to Alan](./contributing.md)
+
+<!---
+TODO proposal
+
+  - Concurrently handle and make http requests without Futures or Promises (hint: do nothing!)
+  - Process large amounts of Array data in parallel without a framework (hint: do nothing!)
+  - Define mutable variables outside and inside event handlers
+  - Implement sequential algorithms that can't use arrays (TODO call out algorithm)
```

Comment:
Maybe make the example on Newton-Raphson's method applied to numerically calculate a derivative or something?

Reply:
Yeah!